fms_extras/models/paged_llama.py

```diff
@@ -742,7 +742,7 @@ def factory(**kwargs):
     max_expected_seq_len=8192,
 )
 
-models.register_model("paged_llama3", "8b", _llama_factory_factory((_8b_llama3_config)))
+models.register_model("paged_llama", "8b", _llama_factory_factory((_8b_llama3_config)))
```

Comment:
better to re-use `_architecture_name` since we already have a constant for this

Reply:
Fixed in latest commit